src/boot/efi/boot.c

```diff
@@ -28,6 +28,7 @@ enum loader_type {
         LOADER_UNDEFINED,
         LOADER_EFI,
         LOADER_LINUX,
+        LOADER_SECUREBOOT,
```

Comment:
secureboot is a weird name. we enroll signature keys here, and i think we should name the entry type like that. i.e. LOADER_SIGNATURE_KEYS?